src/app/components/index/index-footer/index-footer/index-footer.component.html

```diff
@@ -0,0 +1,32 @@
+<footer>
+  <table mdbTable borderless="true" id="facebookTable">
+    <thead>
+      <h4>
```

Comment:
I think it's better to use our fonts/add new instead of using h4

Reply:
Headers already changed to span